index.ts

```diff
@@ -39,8 +40,14 @@ const reverseAutocaptureEvent = (autocaptureEvent: StrippedEvent) => {
 
 const plugin: Plugin<ReplicatorMetaInput> = {
     exportEvents: async (events, { config }) => {
+        const eventsToIgnore = new Set(config.eventsToIgnore.trim() !== "" ? config.eventsToIgnore.split(',').map(event => event.trim()) : null)
```

Comment:
that would make sense but invalidates the condition as it would also be true if the `events_to_ignore` were `undefined`. instead I added an `undefined` check before.